packages/terra-compact-interactive-list/src/subcomponents/Cell.module.scss

```diff
@@ -0,0 +1,18 @@
+// Themes
+@import '../orion-fusion-theme/Cell.module';
+@import '../clinical-lowlight-theme/Cell.module';
+
+:local {
+  .cell {
+    align-items: center;
+    display: flex;
+    font-weight: normal;
```

Comment:
Is the font-size of cell contents supposed to be same for all themes. They are usually smaller for fusion theme.

Reply:
Great catch, I am not sure how `font-weight` it appeared to be there.